### PR TITLE
Feature/stock api

### PR DESCRIPTION
**Pull request by julianzimmermann**

### CODE REVIEW

File: bundles/stock-api/composer.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "fond-of-oryx/stock-api",
+  "description": "The StockApi module provides a REST API for simple product stock updates.",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Marcello Muscara",
```
Comment (SpiderDan98) [inline bundles/stock-api/composer.json:7]: Is Marcello Muscara the correct author?

File: bundles/stock-api/src/FondOfOryx/Zed/StockApi/StockApiDependencyProvider.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function provideApiQueryContainer(Container $container)
+    {
+        $container[static::QUERY_CONTAINER_API] = function (Container $container) {
```
Comment (SpiderDan98) [inline bundles/stock-api/src/FondOfOryx/Zed/StockApi/StockApiDependencyProvider.php:84]: Here you could use a static function or?

File: bundles/stock-api/src/FondOfOryx/Zed/StockApi/StockApiDependencyProvider.php [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function provideAvailabilityQueryContainer(Container $container)
+    {
+        $container[static::AVAILABILITY_QUERY_CONTAINER] = function (Container $container) {
```
Comment (SpiderDan98) [inline bundles/stock-api/src/FondOfOryx/Zed/StockApi/StockApiDependencyProvider.php:98]: Here you could use a static function or?

File: bundles/stock-api/src/FondOfOryx/Zed/StockApi/StockApiDependencyProvider.php [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function provideStockFacade(Container $container)
+    {
+        $container[static::FACADE_STOCK] = function (Container $container) {
```
Comment (SpiderDan98) [inline bundles/stock-api/src/FondOfOryx/Zed/StockApi/StockApiDependencyProvider.php:128]: Here you could use a static function or?

File: bundles/stock-api/src/FondOfOryx/Zed/StockApi/StockApiDependencyProvider.php [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function provideProductFacade(Container $container)
+    {
+        $container[static::FACADE_PRODUCT] = function (Container $container) {
```
Comment (SpiderDan98) [inline bundles/stock-api/src/FondOfOryx/Zed/StockApi/StockApiDependencyProvider.php:142]: Here you could use a static function or?

**Review by SpiderDan98 — APPROVED** [review]

File: bundles/stock-api/composer.json (hunk 1 above)
Comment (julianzimmermann) [reply bundles/stock-api/composer.json:7]: Yes, he made the ported package before

**Pull request merged**